handbook/data-stores/mongodb.md

```diff
@@ -0,0 +1,138 @@
+
+## Single Node Deployments
+
+By default, SystemLink stores all data in databases and the file system hosted on the SystemLink application server. For more reliable data storage and scalability, Ni recommends remote data stores for all production deployments.
+
+### Increasing the index cache size for *single node* deployments
+
+Increase the index cache size to process more data and use the Test Insights or Asset Management modules in a single-node deployment. This avoids high CPU usage that can occur with the default MongoDB memory constraints in SystemLink. NI recommends increasing the available memory for the index to 8GB. Refer to [Sizing a SystemLink Server](https://www.ni.com/documentation/en/systemlink/latest/setup/sizing-a-systemlink-server/) for system requirements to run SystemLink.
+  
+1. With a user with administrator privileges, log into the desktop of the SystemLink app server.
+1. Open **NI SystemLink Server Configuration**.
+1. Navigate to **NoSqlDatabase**.
+1. Under **The NoSqlServer is launched and managed by SystemLink**, adjust the value in **The cache size of the database in Gigabytes (GB)**.
+1. Click **Apply**.
+
+!!!note
+    Refer to [Sizing a SystemLink server](https://www.ni.com/documentation/en/systemlink/latest/setup/sizing-a-systemlink-server/) for guidance on the CPU, memory, and disk needed for SystemLink single node deployments.
+
+## Multi Node Deployments
+
+SystemLink supports three types of multi-node configuration. You can configure SystemLink to connect to a standalone instance, replica sets, or MongoDB Atlas.
+
+NI recommends connecting SystemLink to replica set of three or more MongoDB servers for all production deployments.
+
+!!!important
+    Due to a bug in a 3rd party MongoDB driver, SystemLink cannot connect to MongoDB instances where the MongoDB username provided in the form input or connection string contains either the `-` or the `_` character.
+
+- NI does not support configurations where multiple SystemLink servers use the same MongoDB instance.
+- NI does not support configurations where another production application uses the same MongoDB instance as SystemLink.
+
+!!!note
+    While SystemLink can connect to and use a sharded MongoDB cluster (`mongos`), it will not take advantage of horizontal scaling capabilities enabled by sharded clusters.
+
+When using a stand-alone MongoDB instance, replica set, or Atlas, you have the option to encrypt the data stored within the database. This is built into MongoDB Atlas or requires [MongoDB Enterprise Advanced](https://www.mongodb.com/products/mongodb-enterprise-advanced) for stand-alone or replica sets.
```

Comment:
I may have missed this. I'm using your suggestion with a minor change to point out that these are the options that allow for encryption rather than backup.